bot/common/threads/thread_builder.py

```diff
@@ -198,7 +196,14 @@ def __await__(self):
 
     async def _init_steps(self):
         self.steps = await self.get_steps()
-        self.step = self.find_step(self.steps, self.current_step)
+        # assign the current_step hash to root of the tree if no overriding
+        # current_step is provided when constructing the thread
+        if self.current_step is None:
+            self.current_step = self.steps.hash_
```

Comment:
Why do we need to change this now? Is it related to this change? https://github.com/Govrn-HQ/KevinMalone/pull/160/files#diff-8c706a8372a231a6d172054311fb86d484f30a2cdd2ebb7cc13a48a02c562d3fR16

Reply:
yep, it's related to that change. missed something in that PR, if you create a thread and immediately send it, it runs into a snag.-